### ISSUE TITLE
Takes up my whole bandwidth in background

### ISSUE THREAD

**Issue by maitreyav** [mafintosh/playback#118]
While running in the background it was consuming 0.7 Mbps speed constantly for 2 days. Please check into that. I lost my bandwidth because of this. I feel **very** suspicious regarding the background data activity.

**Comment by imspikey** [comment]
me too :(

**Comment by dralletje** [comment]
It **seems** like it is downloading, but it is downloading from it's own 'server', to show the movie. That is why it seems to drain so much, because it is 'downloading' the whole movie :-) So no, not too suspicious :-/

**Comment by imspikey** [comment]
I dont  think this is suspicious I think Its a bug because its downloading although my list is empty ???!!!

**Comment by maitreyav** [comment]
@dralletje, No, it consumed my 5GB when the list was empty and the movie that I was streaming earlier was of size around 1.1GB and I cleared the list when the application failed to stream it without lag. If it's a bux fix it asap as it will surely gonna raise some eyebrows.

**Comment by dralletje** [comment]
Hey @maitreyav , you are on github too, so why don't you try to help with some samples or with a patch ;)

Can you try it out with the master branch? I don't have it with the master branch, so it may already be fixed. If it is not, please post a gif or something of the "Network" tab of the app and the activity monitors Network tab :-)
